### PR TITLE
Updated and tested as per issue 116.

### PR DESCRIPTION
**Pull request by ChuckKollar**
Issue: https://github.com/x-atlas-consortia/hs-ontology-api/issues/116

### CODE REVIEW

File: dev/app.py [hunk 1]
```diff
@@ -1,34 +1,117 @@
...
+            return ad
+    return None
+
+
+def check_for_valid_application_context() -> None:
+    """
+    Check for a valid auery parameter application_context, and if not found abort with a 400.
+    """
+    application_context = request.args.get('application_context')
+    if application_context != "HUBMAP":
```
Comment (yuanzhou) [inline dev/app.py:80]: Can we make it case-insensitive check to give more tolerance? Some of the API clients use mixed cases.

File: dev/app.py [hunk 2]
```diff
@@ -1,34 +1,117 @@
...
+        abort(400, "A query parameter of application_context=HUBMAP must be specified")
```
Comment (yuanzhou) [inline dev/app.py:81]: When I tested, I got html repose instead of json.
image

**Review by yuanzhou — CHANGES_REQUESTED** [review]
@ChuckKollar two minor changes requested, thanks.

File: dev/app.py (hunk 2 above)
Comment (ChuckKollar) [reply dev/app.py]: Changes to branch have been made.

Comment (yuanzhou) [reply dev/app.py]: Thanks @ChuckKollar !

**Pull request merged**